### DISCUSSION TITLE
Oceanbase Diagnostic Tool 1.0.0正式发布

### DISCUSSION THREAD

**Discussion by Teingi** [oceanbase/obdiag#11] [Announcements]
## Program specification
The OceanBase Diagnostic Tool is a functional upgrade of the original OceanBase Diagnosis Gather, which forms a unified black screen command style and is easy to use.

## Features
- Support to gather oceanbase logs from oceanbase machines
- Support to gather AWR report
- Support to gather Host information
- Support to gather clog/slog
- Support to gather perf
- Support to gather ParalleSQL information
- Support to gather obproxy log from obproxy machines

## 概要说明
OceanBase诊断工具(OceanBase Diagnostic Tool) 是原诊断信息收集工具 (OceanBase Diagnosis Gather)的功能升级版，形成了统一的黑屏命令风格，简单易用。

## 特性
- 支持从oceanbase主机上收集OB的日志
- 支持收集AWR报告
- 支持收集主机信息
- 支持收集(clog、slog解析后的日志)
- 支持收集perf信息（扁鹊图、perf火焰图、pstack火焰图）
- 支持收集并行SQL的执行详情信息
- 支持收集OBPROXY的日志

**Comment by Teingi** [comment]
欢迎大家使用.

**Comment by plmqwsa** [comment]
Very useful ！！

**Reply by Teingi** [reply]
release: https://github.com/oceanbase/oceanbase-diagnostic-tool/releases
